fedimintd/src/fedimintd.rs

```diff
@@ -249,6 +287,14 @@ impl Fedimintd {
             .with_label_values(&[fedimint_version, code_version_hash])
             .set(fedimint_core::time::duration_since_epoch().as_secs() as i64);
 
+        if env::var(FM_BIND_API_ENV).is_ok() {
+            bail!(
+                "{} environment variable was removed and replaced with two separate: {} and {}. Please unset it.",
+                FM_BIND_API_ENV,
+                FM_BIND_API_WS_ENV,
+                FM_BIND_API_IROH_ENV,
+            );
+        }
```

Comment:
That's elegant!

Reply:
Is it? I kind of thought it's clunky, but will protect people from scratching their heads. :D

Reply:
I was thinking of how to allow both old and new env vars to co-exist and that would have been way clunkier xD just failing and warning about it didn't occur to me